tests/login.test.js

```diff
@@ -0,0 +1,55 @@
+import LoginPage from '../pages/login.page';
+
+describe('DigitalBank', () => {
+
+    //example test
+    it('Should display login page when browsing to the route', async () => {
+      await LoginPage.open('login');
+      expect(browser).toHaveUrlContaining('login');
+    });
+
+    it('Should log into demo account when valid data is entered', async () => {
+     
+      await LoginPage.open('login');
+      await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+      await LoginPage.clickElement(LoginPage.signInButton);
+      expect(browser).toHaveUrlContaining('home');
+
+    });
+
+    it('Should log into demo account when using enter key', async () => {
+      await LoginPage.open('login');
+      await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+      let loginButton = await $('[id="submit"]');
```

Comment:
You are mixing 2 levels of abstraction here (pageObjects methods and raw web elements located through selectors). Ideally, any function should work at the same level of abstraction to make it easier to mantain and understand. Consider implementing this loginButton as a pageObject attribute (such as username input and password), and access it through one of the logIn methods proposed in a previous comment. That way you encapsulate all the complexity regarding how to interact with page's element at pageObject level (making the process transparent for the test)